frontend/src/lib/api/destructionListsItem.ts

```diff
@@ -33,6 +33,7 @@ export async function listDestructionListItems(
         page_size?: number;
         "item-processing_status"?: ProcessingStatus;
         "item-status": DestructionListItemStatus; // TODO ?
+        "item-order_review_ignored"?: string;
```

Comment:
item-order_review_ignored (both hyphens and underscores) seems odd, is this as such provided by the backend?